AndroidSDK/src/com/leanplum/internal/Request.java

```diff
@@ -198,24 +199,15 @@ private static void saveRequestForLater(Map<String, Object> args) {
       SharedPreferences preferences = context.getSharedPreferences(
```

Comment:
does the whole method need to be synchronized now? I think only saving new uuid needs to be blocking?